src/fmu/dataio/_model/schema.py

```diff
@@ -134,18 +129,18 @@ def _deprecation_warning(self) -> UnsetContent:
         return self
 
 
-class UnsetAnyContent(data.AnyData):
-    root: UnsetContent  # type: ignore
+class InternalAnyData(data.AnyData):
+    root: InternalUnsetData  # type: ignore
```

Comment:
I think @tnatt should probably weigh in on this. I think the naming here is probably the right thing to do, but she has the best understanding of the `Unset` stuff